diesel/src/associations/mod.rs

```diff
@@ -356,7 +348,7 @@ impl<'a, T: HasTable> HasTable for &'a T {
 ///
 /// ### Deriving
 ///
-/// This trait can be automatically derived using `diesel_codegen` by adding
+/// This trait can be automatically derived using `diesel_derives` by adding
```

Comment:
If this is automatically included by diesel now, we should remove the

> using `diesel_derives`

Reply:
There is much more documentation wrong/incorrect/broken after this. I think it is more meaningful to first finish the actual implementation and fix all those documentation things afterwards.